spec/soda_machine_spec.rb

```diff
@@ -51,4 +54,23 @@
     end
   end
 
+  describe "#find_cheapest_soda", { current_inventory_count: true } do
+    it "returns the cheapest soda item in the inventory" do
+      expect(soda_machine.find_cheapest_soda).to eq([second_pepsi, pepsi])
```

Comment:
"cheapest" to me means some absolute thing, not a pair of things. i'm surprised that this returns more than one item.